modules/nomad-clients/scripts/setup_client.tftpl.sh

```diff
@@ -87,8 +87,14 @@ search ap-south-1.compute.internal
 EOF
 }
 
+# Increase the file limit
+modify_nomad_systemd_config() {
+  sudo sed -i 's/LimitNOFILE=65536/LimitNOFILE=900000/' /lib/systemd/system/nomad.service
```

Comment:
how about instead of hardcoding it to `900000`, we set a variable with that default value and allow users to configure it? we can add conditions to check if the value is > `65535`

Reply:
Yes, we can do that. I can add it as a terraform variable with a default of `900000`.